docs/rfcs/39-promote-abyss-w-as-go-storage-reviewer.md

```diff
@@ -0,0 +1,37 @@
+- Author: JinnyYi <github.com/JinnyYi>
+- Start Date: 2021-08-20
+- RFC PR: [beyondstorage/community#39](https://github.com/beyondstorage/community/issues/39)
+- Tracking Issue: N/A
+
+# RFC-39: Promote abyss-w as go-storage Reviewer
+
+## Background
+
+[abyss-w] is an active contributor of go-storage and related services since July. Due to her contributions, we nominate her as go-storage contributor in [#21](https://github.com/beyondstorage/community/pull/21). Since then, she has continued to improved her personal skills and project understanding, continued to contribute to community.

```

Comment:
```suggestion
[abyss-w] is an active contributor of go-storage and related services since July. Due to her contributions, we nominate her as go-storage contributor in [#21](https://github.com/beyondstorage/community/pull/21). Since then, she has continued to participate in community communication and contribute to the project.
```

I think we should not judge "improved her personal skills and project understanding" in nomination? In other words, nomination is based on contribution and commitment, instead of competence and improvement?

Reply:
Others LGTM